src/components/PeopleFilters.tsx

```diff
@@ -1,18 +1,77 @@
+import { useCallback } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import cn from 'classnames';
+
+import { SearchParams, getSearchWith } from '../utils/searchHelper';
+import { SearchLink } from './SearchLink';
+
 export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const FEMALE = searchParams.get('sex') === 'f';
+  const MALE = searchParams.get('sex') === 'm';
```

Comment:
Use a `camelCase` for these variables. They are created when your component renders, so they are not constants.

Don't use magic values like `'f'` or `'m'`.